App/Navigation/AppNavigation.js

```diff
@@ -8,6 +8,7 @@ import Login from '../Containers/Login'
 
 import { Colors } from '@Themes'
 const DrawerStack = createDrawerNavigator({
+  TempScreen: { screen: Login }
```

Comment:
The next error was around the fact that you need at least one screen in a navigator. I gave it the login one just to get through the error.